src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/notification/VisitNotificationEvent.kt

```diff
@@ -26,7 +26,7 @@ class VisitNotificationEvent(
   val type: NotificationEventType,
 
   @Column(nullable = true)
```

Comment:
I think this will need the name = "event_description" added

Reply:
Aled suggested I rename to description, so this won't be needed now ^